Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -203,6 +217,7 @@ public static void VerifyID(string deviceId, Action<LootLockerVerifyResponse> on
         /// </summary>
         /// <param name="deviceId">The ID of the current device the player is on</param>
         /// <param name="onComplete">onComplete Action for handling the response of type LootLockerSessionResponse</param>
+        [Obsolete("DEPRECATED: Please use the StartSession method for the platform you're on. For Android use Guest Session. For iOS use Apple Session")]
```

Comment:
I would add; if you are unsure, use StartGuestSession(). Mostly because of the Zyger video.

Reply:
Fixed